pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -516,9 +516,13 @@ func validateNodeGroupIAM(iam *NodeGroupIAM, value, fieldName, path string) erro
 
 // ValidateManagedNodeGroup validates a ManagedNodeGroup and sets some defaults
 func ValidateManagedNodeGroup(ng *ManagedNodeGroup, index int) error {
-	if ng.AMIFamily != NodeImageFamilyAmazonLinux2 {
-		return fmt.Errorf("only %s is supported for Managed Nodegroups", NodeImageFamilyAmazonLinux2)
+	switch ng.AMIFamily {
+	case NodeImageFamilyAmazonLinux2, NodeImageFamilyBottlerocket:
+
+	default:
+		return errors.Errorf("%q is not supported for managed nodegroups", ng.AMIFamily)
 	}
```

Comment:
does this need to be a switch statement? looks weird seeing an empty `case`

Reply:
It can be an `if` statement but a `switch` is more concise when comparing against the same value. Ubuntu support will add another value to the `case` clause.